lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100vw;
```

Comment:
ooof good call http://caniuse.com/#search=vw

_can you imagine a world without IE..._

ill flip it to 100%